tests/js/core/widget/core/Drawer/test-data/sample.html

```diff
@@ -0,0 +1,27 @@
+<div class="ui-page ui-page-active" id="page-1">
+	<div class="ui-contnent">
+		<div class="ui-drawer" data-position="left" id="drawer-1" data-drag-edge="50">
```

Comment:
question.
what is data-drag-edge? and why value 50 fixed?

Reply:
I copied this `html` from examples, but this option is defined in Drawer widget (https://github.com/Samsung/TAU/blob/dc24f0903ccda2c8c0fb34b4f9696ea9c55816c2/src/js/core/widget/core/Drawer.js#L126)